protocols/peralta_yahya_lab/README.md

```diff
@@ -0,0 +1,46 @@
+# Flow Cytometry Preparation
+
+### Author
+[Opentrons](https://opentrons.com/)
+
+### Partner
+Georgia Institute of Technology
+
+## Categories
+* Sample Prep
+	* Distribution
+
+
+## Description
+This protocols allows the robot to transfer media, chemical, and cells into wells B2-G11 of up to 4 different plates. 
+
+Steps:  
+1. Fills the wells B2-G11 of the target 96-well plates (P2, P4, P5, and P6) with 182 uL of media from a 50 mL tube using a P300_Single pipette.
```

Comment:
We had decided previously that these steps will go into the `process` section. Description itself should be more scientific such as the following (copied from [this ](http://protocols.opentrons.com/protocol/in_gel_digest)protocol):

Prepares proteins harbored inside gels for overnight protease digestion to produce peptides for downstream mass spectrometry analysis. This protocol is designed for SDS-reducing gels (PAGE). Through a series of steps, disulfide bridges between cysteines are reduced and the sulfur atoms are capped to prevent the reformation of the disulfide bonds. The two wash steps thereafter removes staining as well as as excess DTT, acrylamide and ammonium bicarbonate, preparing the gels for the uptake of protease(s) in a pH 8 solution. Protocol set for 20 samples.